bootstrap/kubeadm/types/v1beta2/bootstraptokenstring_test.go

```diff
@@ -69,20 +67,15 @@ func TestUnmarshalJSON(t *testing.T) {
 		t.Run(rt.input, func(t *testing.T) {
 			newbts := &BootstrapTokenString{}
 			err := json.Unmarshal([]byte(rt.input), newbts)
-			if (err != nil) != rt.expectedError {
-				t.Errorf("failed BootstrapTokenString.UnmarshalJSON:\n\texpected error: %t\n\t  actual error: %v", rt.expectedError, err)
-			} else if !reflect.DeepEqual(rt.bts, newbts) {
-				t.Errorf(
-					"failed BootstrapTokenString.UnmarshalJSON:\n\texpected: %v\n\t  actual: %v",
-					rt.bts,
-					newbts,
-				)
-			}
+			g.Expect(err != nil).To(Equal(rt.expectedError))
```

Comment:
I think we can be a bit clearer with a bit of extra verbosity here:
```suggestion
			if rt.expectedError {
				g.Expect(err).To(HaveOccurred())
			} else {
				g.Expect(err).NotTo(HaveOccurred())
			}
```

Reply:
done